utility/src/main/java/org/oppia/android/util/parser/ImageLoader.kt

```diff
@@ -2,7 +2,10 @@ package org.oppia.android.util.parser
 
 import android.graphics.Bitmap
 import android.graphics.drawable.PictureDrawable
-import com.bumptech.glide.load.Transformation
+
+enum class ImageTransformation {
+  BLUR
+}
```

Comment:
Please add KDocs for the enum & BLUR to explain what they are & how they should be used.

Reply:
Done. Feel free to add suggestions for the KDoc.